crates/gateway-client/src/builder.rs

```diff
@@ -191,9 +193,8 @@ impl<'a> Request<'a, stage::Params> {
         self.add_param("classHash", &class_hash.0.to_hex_str())
     }
 
-    pub fn with_casm_hash(self, hash: CasmHash) -> Self {
-        pathfinder_common::version_check!(Integration < 0 - 11 - 0, "Confirm parameter name");
-        self.add_param("classHash", &hash.0.to_hex_str())
+    pub fn with_sierra_hash(self, class_hash: SierraHash) -> Self {
+        self.add_param("classHash", &class_hash.0.to_hex_str())
```

Comment:
How is this different than `with_class_hash`?

I mean _conceptually_ a Sierra class hash is just a class hash.

Reply:
Well, it's not. I think we should remove the types: `SierraHash` and `CasmHash`.